homeassistant/components/http.py

```diff
@@ -386,6 +411,38 @@ def is_trusted_ip(self, remote_addr):
         return any(ip_address(remote_addr) in trusted_network
                    for trusted_network in self.hass.http.trusted_networks)
 
+    def wrong_login_attempt(self, remote_addr):
+        """Registering wrong login attempt."""
+        if not self.is_trusted_ip(remote_addr) \
```

Comment:
Yes, you're correct, trusted IP get `authenticated = True` -- removing this check